README.md

```diff
@@ -27,12 +27,57 @@ The standardized MAFs generated by `standardize_mutation_data.py` will be writte
 
 **Sequence Source** ( `-s | --sequence-source` ) : The sequence source is a required field for a standard MAF as defined by the TCGA project. Accepted values are either WGS and WXS. For GENIE, WXS is used since GENIE samples are sequenced with targeted gene panels.
 
-**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts are located:
+**Annotation Suite Scripts Directory** ( ` -p | --annotation-scripts-home` ) : needs to point to the directory where the packaged Genome Nexus Annotation Pipeline JAR and annotation suite python scripts and data files are located:
 
-- merge_mafs.py
-- standardize_mutation_data.py
-- annotator.jar
+- merge_mafs.py (located in this repository)
+- standardize_mutation_data.py (located in this repository)
+- annotator.jar - The genome nexus annotation pipeline (See how to build the jar file [here](https://github.com/genome-nexus/genome-nexus-annotation-pipeline))
+- application.properties (optional) - custom properties for the annotation pipeline. The common use case is to specify custom genome nexus url with `genomenexus.base` property.
```

Comment:
Is it true that just putting a file named application.properties into the scripts directory will cause the running of the scripts to use those application properties? I thought that properties files need to be built into the software .. in other words, the annotator.jar should already contain the application.properties file in order for it to be effective.

Also, I forget ... it is necessary for the annotator.jar file to be present in the same directory as the scripts? Maybe we used that convention for the annotation-tools development in order to keep things simpler for AACR Genie when they ran the tool (which is fine). But in our own environment we normally keep the compiled jar files in a separate (lib) directory, so I wonder what the case is here.

Reply:
Ok, yes, the line you reference seems to require the .jar file to be in the same directory as the scripts.  In our own environment we do not regularly run annotation-tools, and it is mainly our data curators who use annotator.jar. When they use annotator.jar I believe they run bash scripts which our team wrote that execute the .jar code in a command like this:
$JAVA_HOME/bin/java -jar $JAR_LIB_PATH/annotator.jar -i uniprot -r -f $PROJECT_DATA_HOME/data_mutation.maf  -o $PROJECT_DATA_HOME/data_mutation_annotated.maf > $logfile_path

but I think for annotation_tools we don't have to worry about this.